packages/@okta/vuepress-site/documentation/index.md

```diff
@@ -45,6 +45,9 @@ languages:
 sections:
```

Comment:
@bretterer, in the old site, this page would get automatically updated: you just had to create a new page in the 'use cases' or 'API reference' areas, and the link would be added to this page.

Reply:
Changing that was intentional. I don't want pages to magically show up on the index anymore. We are going to curate them even more strongly in the future to make sure the most valuable stuff is shown on the home page.